app/classifier/tasks/survey/choice.cjsx

```diff
@@ -44,15 +44,19 @@ ImageFlipper = React.createClass
 module.exports = React.createClass
   displayName: 'Choice'
 
+  propTypes:
+    annotation: React.PropTypes.object.isRequired
```

Comment:
Looks like [`filters` is also a generic object](/zooniverse/Panoptes-Front-End/blob/master/app/classifier/tasks/survey/index.cjsx#L42).